test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityPCLTests.cs

```diff
@@ -32,6 +32,115 @@ public void CompatibilityPCL_Net()
             Assert.False(compat.IsCompatible(framework2, framework1));
         }
 
+        [Fact]
+        public void CompatibilityPCL_WithOneTfm()
+        {
+            var framework1 = NuGetFramework.Parse("net45");
+            var framework2 = NuGetFramework.Parse("portable-net45");
+
+            var compat = DefaultCompatibilityProvider.Instance;
+
+            Assert.True(compat.IsCompatible(framework1, framework2));
+            Assert.False(compat.IsCompatible(framework2, framework1));
+        }
+
+        [Fact]
+        public void CompatibilityPCL_ZeroTfmIntoNonPclProject()
+        {
+            var framework1 = NuGetFramework.Parse("net45");
+            var framework2 = NuGetFramework.Parse("portable");
+
+            var compat = DefaultCompatibilityProvider.Instance;
+
+            Assert.False(compat.IsCompatible(framework1, framework2));
+            Assert.False(compat.IsCompatible(framework2, framework1));
+        }
+
+        [Fact]
+        public void CompatibilityPCL_ZeroTfm()
+        {
+            var framework1 = NuGetFramework.Parse("portable");
+            var framework2 = NuGetFramework.Parse("portable");
+
+            var compat = DefaultCompatibilityProvider.Instance;
+
+            Assert.True(compat.IsCompatible(framework1, framework2));
+            Assert.True(compat.IsCompatible(framework2, framework1));
+        }
+
+        [Theory]
+        // supported profiles
+        [InlineData("portable-net45+netcore45", "netstandard6.0", false)]          // Profile7 -> netstandard 1.0 - 1.1
```

Comment:
The portable mappings should not be bidirectional. portable-blah should not pull in netstandard packages but vice versa should work. Remember this is there only for transition.